packages/core/src/internal-components/panel/panel.element.ts

```diff
@@ -32,13 +31,15 @@ export const CdsInternalPanelTagName = 'cds-internal-panel';
  * @cssprop --box-shadow
  */
 export class CdsInternalPanel extends LitElement {
+  @id() private panelId: string;
+
   static get styles() {
     return [baseStyles, styles];
   }
 
   render() {
     return html`
-      <div class="private-host">
+      <div class="private-host" role="region" aria-labelledby="${this.panelId}">
```

Comment:
I don't think this is what @coryrylan was suggesting...

Reply:
Yea I forgot to remove that one. Thank you for reminding me!

Reply:
@coryrylan I remember we actually talked about the `role="region"` should be part of the `div.private-host` so isn't it what we want as part of the `panel`?  If this case is the correct one then doesn't this means we have to pass down the `aria-labelledby` attribute that will be set on the `cds-card` to the `div.private-host`?

Reply:
What you have currently on the PR looks correct. With the role on the host its remains in the light DOM and the a11y is preserved.

```html
<!-- App code -->
<cds-card aria-describedby="heading">
  <h2 id="heading">heading</h2>
</cds-card>

<!-- Rendered output code -->
<cds-card role="region" aria-describedby="heading">
  <h2 id="heading">heading</h2>
</cds-card>
```